client/templates/hangout/hangout-item.html

```diff
@@ -17,7 +17,9 @@
       </label>
       <ul class="status">
         <li><i class="fa {{getType type}}"></i></li>
-        <li><label>{{getDate this}}</label></li>
+        <li><label>{{getDate this}} |  <a class="clone-hangout">
```

Comment:
To add to this. I am receiving the following error when invoking the method `cloneHangout`

**Exception while invoking method 'cloneHangout' Error: Match error: Expected string, got object in field user_id**

This is because there's no user logged in! So the call actually fails and you are not able to copy/create the hangout.